mlflow/tracking/fluent.py

```diff
@@ -1167,6 +1169,104 @@ def pagination_wrapper_func(number_to_get, next_page_token):
     )
 
 
+def search_registered_models(
+    max_results: Optional[int] = None,
+    filter_string: Optional[str] = None,
+    order_by: Optional[List[str]] = None,
+) -> List[RegisteredModel]:
+    """
+    Search for registered models in backend that satisfy the filter criteria.
+
+    :param filter_string: Filter query string
+        (e.g., ``"name = 'a_model_name' and tag.key = 'value1'"``),
+        defaults to searching for all registered models. The following identifiers, comparators,
+        and logical operators are supported.
+
+        Identifiers
+          - ``name``: registered model name.
+          - ``tags.<tag_key>``: registered model tag. If ``tag_key`` contains spaces, it must be
+            wrapped with backticks (e.g., ``"tags.`extra key`"``).
+
+        Comparators
+          - ``=``: Equal to.
+          - ``!=``: Not equal to.
+          - ``LIKE``: Case-sensitive pattern match.
+          - ``ILIKE``: Case-insensitive pattern match.
+
+        Logical operators
+          - ``AND``: Combines two sub-queries and returns True if both of them are True.
+
+    :param max_results: If passed, specifies the maximum number of models desired. If not
+                        passed, all models will be returned.
+    :param order_by: List of column names with ASC|DESC annotation, to be used for ordering
+                     matching search results.
+    :return: A list of :py:class:`mlflow.entities.model_registry.RegisteredModel` objects
+            that satisfy the search expressions.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Get search results filtered by the registered model name
+        model_name="CordobaWeatherForecastModel"
```

Comment:
Same here https://github.com/mlflow/mlflow/pull/7428/files#r1035809582

Reply:
I would like to make this docstring consistent with https://github.com/mlflow/mlflow/blob/893fc133ca0a2043ffb8ae639255a493210225f7/mlflow/tracking/_model_registry/client.py#L95 
So I think it is better to leave it as it is.  
WDYT?

Reply:
Got it. I don't have a strong preference here. Let's leave it as it is.